cd/application-deployment/dev/vaec-api-task-definition.json

```diff
@@ -233,18 +225,6 @@
         }
       ],
       "secrets": [
-        {
-          "name": "COMP_AND_PEN_SERVICE_ID",
-          "valueFrom": "arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/dev/notification-api/comp-and-pen/service-id"
-        },
-        {
-          "name": "COMP_AND_PEN_TEMPLATE_ID",
-          "valueFrom": "arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/dev/notification-api/comp-and-pen/template-id"
-        },
-        {
-          "name": "COMP_AND_PEN_SMS_SENDER_ID",
-          "valueFrom": "arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/dev/notification-api/comp-and-pen/sms-sender-id"
-        },
```

Comment:
Question: these are used in `app/config.py` - are these actually read from the celery task definition?

Reply:
These still need to be in the config file - They are used by the celery code path and have not been removed from the celery task definition. I have removed them from the API container.

Reply:
If os.getenv('COMP_AND_PEN_SMS_SENDER_ID') is called in the `notificiation-api`, it will return None. If it's called in the `celery` container - it will be made available in `current_app.config `and then accessed by `send-scheduled-comp-and-pen-sms`

Reply:
wait - I might be wrong about this. Hang on doing some testing.